packages/grid/x-data-grid-premium/src/hooks/features/aggregation/useGridAggregationPreProcessors.tsx

```diff
@@ -121,18 +122,41 @@ export const useGridAggregationPreProcessors = (
       });
 
       if (availableAggregationFunctions.length === 0) {
-        return initialValue;
+        return columnMenuValue;
       }
 
-      return [
-        ...initialValue,
-        <Divider />,
-        <GridAggregationColumnMenuItem
-          column={column}
-          label={apiRef.current.getLocaleText('aggregationMenuItemHeader')}
-          availableAggregationFunctions={availableAggregationFunctions}
-        />,
-      ];
+      const aggregationItemProps = {
+        column,
+        label: apiRef.current.getLocaleText('aggregationMenuItemHeader'),
+        availableAggregationFunctions,
+      };
+
+      const slot: GridColumnMenuLookup['slot'] = 'aggregation';
+
+      const aggregationItem = columnMenuValue.items.some(({ displayName }) =>
+        displayName?.includes('GridFilterMenuSimple'),
+      )
+        ? {
+            slot,
+            displayName: 'GridAggregationColumnMenuSimpleItem',
+            component: <GridAggregationColumnMenuSimpleItem {...aggregationItemProps} />,
```

Comment:
`displayName` was just used to differentiate between the Simple and Default menu in Pro and Premium packages so that the proper component is injected. But now, I am moving towards configurable component slots for Pro/Premium menu items. `displayName` will be removed eventually.